bcda/models/models_test.go

```diff
@@ -288,19 +288,28 @@ func (s *ModelsTestSuite) TestGroupStructs() {
 		Resource{ID: "xxx", Name: "BCDA API", Scopes: []string{"bcda-api"}},
 		Resource{ID: "eft", Name: "EFT CCLF", Scopes: []string{"eft-app:download", "eft-data:read"}},
 	}
-	systems := []System{
-		System{ClientID: "4tuhiOIFIwriIOH3zn", SoftwareID: "4NRB1-0XZABZI9E6-5SM3R", ClientName: "ACO System A", ClientURI: "https://www.acocorpsite.com"},
-	}
-	groupData := GroupData{Name: name, Users: users, Scopes: scopes, Resources: resources, Systems: systems}
+	groupData := GroupData{Name: name, Users: users, Scopes: scopes, Resources: resources}
```

Comment:
Where is the systems object of the group in the json? Even though systems have their own separate table, they will still be in the json request that the API has to deal with, need to be decoded properly, and most likely be stored in the json column. In other words, we will want to keep the last group definition we got from the user via the API. 

I think this test would be easier to follow if you embedded the json out of the tech spec as a string, converted it to bytes, unmarsheled it into a struct and carried on from there. That is what will happen in the API code.

Reply:
`Group` is what is being stored in the database.  so `Systems` are an array in the `Group` json

Reply:
it's there if you print a `Group`

Reply:
>I think this test would be easier to follow if you embedded the json out of the tech spec as a string, converted it to bytes, unmarsheled it into a struct and carried on from there. That is what will happen in the API code.

That's more complex now because we decided to go with the jsonb field.  so it is actually different than the spec already.

Reply:
but i will try it with the `data` field we added